### PR TITLE
Add documentation for Accounts

### PR DESCRIPTION
**Pull request by sameerag**
Docs for Accounts usage:
* `getAccounts()` and `getAccountByUsername()` usage
* Guidance on multiple accounts use cases

Note:
Part of the doc will be replaced with `typedocs` links when `msal-common` typedocs are available. cc @jo-arroyo

### CODE REVIEW

File: lib/msal-common/docs/Accounts.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Accounts in MSAL JS
+
+MSAL JS libraries (`msal-browser` and `msal-node`) support both single account and multiple accounts scenarios in java script applications. An `account` object is standardized across the libraries:
```
Comment (DarylThayil) [inline lib/msal-common/docs/Accounts.md:3]: javascript one word.

File: lib/msal-common/docs/Accounts.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+* We provide a public API `getAllAccounts()` which lists all the accounts currently in the cache. An application must choose an account to acquire tokens silently.
+* `msal-browser` provides a public API `getAccountByUsername()` for an application to retrieve a specific account based on `username`
+
+Sample usage is as below:
+
+``` javascript
+
+function handleResponse(resp) {
+    if (resp !== null) {
```
Comment (DarylThayil) [inline lib/msal-common/docs/Accounts.md:26]: might want an example that doesn't have `showWelcomeMessage`

File: lib/msal-common/docs/Accounts.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            console.error(error);
+        }
+    });
+}
+```
+
+## Notes
+
+* The current msal-browser default [sample](../../samples/msal-browser-samples/VanillaJSTestApp2.0) has a working single account scenario.
+* If you have a multiple accounts scenario, please modify the sample (in `handleResponse()`) to list all cached accounts and choose a specific account
```
Comment (DarylThayil) [inline lib/msal-common/docs/Accounts.md:62]: which sample?

Comment (sameerag) [reply lib/msal-common/docs/Accounts.md]: Updated (added a link to the exact file this change needs to go in the sample)

File: lib/msal-common/docs/Accounts.md (hunk 2 above)
Comment (sameerag) [reply lib/msal-common/docs/Accounts.md:24]: removed all `showWelcomeMessage()` calls

File: lib/msal-common/docs/Accounts.md (hunk 1 above)
Comment (sameerag) [reply lib/msal-common/docs/Accounts.md]: modified

File: lib/msal-common/docs/Accounts.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    if (resp !== null) {
+        username = resp.account.username;
+        ...
+    } else {
+        // need to call getAccount here?
+        const currentAccounts = myMSALObj.getAllAccounts();
+        if (currentAccounts === null) {
+            return;
+        } else if (currentAccounts.length > 1) {
+            // Add choose account code here
```
Comment (jmckennon) [inline lib/msal-common/docs/Accounts.md:33]: Yeah, my only call out that I brought up in the other doc was this here. I think we should provide a baseline multiple accounts case that we can point people to--either here or in a "multiple accounts" sample

Comment (pkanher617) [reply lib/msal-common/docs/Accounts.md:33]: Agreed, we will work to add some samples showing multiple account scenarios in future PRs.

**Pull request merged**